supabase/migrations/20231118013948_create_initial_tables.sql

```diff
@@ -38,7 +38,8 @@ create table
 create table
   tags (
     id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
-    name varchar(255) not null unique
+    name varchar(255) not null unique,
+    user_id UUID REFERENCES users(id)
```

Comment:
Each user can create tags, so I'd say the unique constraint are supposed to have both user_id and name columns.

```suggestion
    name VARCHAR(255) NOT NULL,
    user_id UUID REFERENCES users(id),
    
    CONSTRAINT unique_name_by_user UNIQUE(name, user_id)
```

Reply:
Thank you 🙏 
I fixed it https://github.com/tabi-memo/backend/pull/17/commits/a49e8b6f3cde83a762dd3014996cd45ceaace002